src/external/rawspeed/RawSpeed/Common.h

```diff
@@ -144,7 +144,7 @@ inline uint32 getThreadCount()
 {
 #ifdef NO_PTHREAD
   return 1;
-#elif WIN32
+#elif defined(_WIN32)
```

Comment:
Changes to everything under `external` need to be pushed upstream so keeping them to a minimum is better. Thus my question, is there a problem with the old version of the check?